opennms-provision/opennms-provisiond/src/main/java/org/opennms/netmgt/provision/service/DefaultProvisionService.java

```diff
@@ -249,6 +251,9 @@ public void updateNode(final OnmsNode node, String rescanExisting) {
         m_nodeDao.update(dbNode);
         m_nodeDao.flush();
 
+        if(!prevLocation.equals(currentLocation)) {
+            accumulator.sendNow(EventUtils.createNodeLocationChangedEvent("Provisiond", dbNode.getId(), dbNode.getLabel(), prevLocation, currentLocation));
```

Comment:
Is there a constant we can use instead the `"Provisiond" literal? (Similar bellow too)